MapleServer2/PacketHandlers/Game/UserSyncHandler.cs

```diff
@@ -42,25 +43,31 @@ public override void Handle(GameSession session, PacketReader packet)
 
             Packet syncPacket = SyncStatePacket.UserSync(session.FieldPlayer, syncStates);
             session.FieldManager.BroadcastPacket(syncPacket, session);
-            if (IsCoordSafe(session, syncStates[0].Coord.ToFloat()))
+
+            CoordF closestBlock = syncStates[0].Coord.ToFloat().ClosestBlock();
+            closestBlock.Z -= 150; // Get block under player
```

Comment:
150 should be made into a constant

Reply:
@AlanMorel @angelotadeucci 
I would create a method somewhere, maybe in `Coord`, which returns the `BlockSize`, which is a constant of 150.

Reply:
Actually I would prefer to have a `Block` class which implement the `BlockSize` method and I'd also move the `ClosestBlock` method there